docs/src/modules/components/DiamondSponsors.js

```diff
@@ -109,9 +103,7 @@ export default function DiamondSponsors() {
                 '& a': {
                   borderColor: 'divider',
                   '&:hover': {
-                    color: 'primary.300',
-                    borderColor: 'primaryDark.600',
-                    backgroundColor: alpha(theme.palette.primaryDark[700], 0.4),
+                    backgroundColor: theme.palette.primaryDark[700],
```

Comment:
Any reason to remove the background opacity?

Reply:
I'm outdoor, I could not see the difference, but no reason to revert.

Reply:
I'm happy to change it, just wanted to know if there was a reason!